client/src/components/Category/index.js

```diff
@@ -68,21 +69,11 @@ class Category extends Component {
           return object;
         });
 
-        const detales = [
-          {
-            id: 1,
-            title: category,
-            background:
-              'https://t4.ftcdn.net/jpg/01/66/07/21/500_F_166072192_CCSQGkwDcjHm2IEK8u2DYBjE5IyEG5NI.jpg',
-            text: `Lorem ipsum dolor sit amet, donec augue porta ultricies magna,
-              mus habitant amet sollicitudin, ad ante diam sed etiam tristique,
-              pellentesque erat. Dui enim vel ac posuere sit eros. Mus et viverra quisque,
-              maiores aliquet fusce pellentesque feugiat, ac sed, gravida est magna. Non quis,`,
-          },
-        ];
+        const details = CategoryDetailsData.find(x => x.name.includes(str));
```

Comment:
@asmaa-ezz Would this still work if the find call checked exact equality, i.e. `x => x.name === str`?  I wonder if the includes() call might give unexpected results sometimes, if there are category names in the database that are similar (like "Education" and "Physical Education", includes() might match the wrong one).